cypress/integration/pages/frontPage/testsForCanonicalOnly.js

```diff
@@ -11,6 +11,28 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({ service }) => {
   if (Cypress.env('APP_ENV') === 'local') {
     runCanonicalAdsTests({ service });
   }
+  describe(`No testsToAlwaysRunForCanonicalOnly to run for ${service}`, () => {
+    describe(`US Election Banner`, () => {
+      it('should have US Election banner for Hindi, Arabic, Portuguese, Mundo, Persian, Russian', () => {
+        const servicesWithBanner = [
```

Comment:
Instead of this, let's use the toggle logic
```suggestion
        cy.getToggles(config[service].name);
         cy.fixture(`toggles/${config[service].name}.json`).then(toggles => {
            const usElectionBannerEnabled = path(
              ['us2020ElectionBanner', 'enabled'],
              toggles,
            );
            if (usElectionBannerEnabled) {
            ...
        }
```